ONSdigital/eq-author-app
# EAR-1962 - Unlink prepop schema

### What is the context of this PR?

https://jira.ons.gov.uk/browse/EAR-1962

### How to review

- Create a questionnaire, go to the 'ONS Dataset' tab (add the feature flag 'dataset' to enable this)
- View the export schema and check that there is no element called 'prepopSchema'
- Select a survey ID from the dropdown and from the displayed table, click on the 'Link' button
- The page will refresh and display the linked dataset details along with the survey ID.
- View the export schema and there must be an element called 'prepopSchema' which contains the linked dataset details
- Click on the 'Unlink dataset' button, a confirmation modal will appear and click on the 'Unlink' button
- The dataset will be unlinked and the page will refresh back to its original state with the surveyID dropdown
- View the export schema and check that there is no element called 'prepopSchema'

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
